library/Rattletrap/Attribute/LoadoutOnline.hs

```diff
@@ -7,33 +7,43 @@ import qualified Data.Binary.Bits.Get as BinaryBit
 import qualified Data.Binary.Bits.Put as BinaryBit
 
 newtype LoadoutOnlineAttribute = LoadoutOnlineAttribute
-  { loadoutAttributeValue :: [[(Word32, CompressedWord)]]
+  { loadoutAttributeValue :: [[(Word32, Either CompressedWord Word32)]]
```

Comment:
This works, but the `Either` is annoying to work with and would be a breaking change.